website/docs/getting-started/setup/upgrade-to-business-edition/README.md

```diff
@@ -1,29 +1,42 @@
+---
+description: Follow the steps to upgrade the Appsmith installation to a Business Edition.
+---
 # Upgrade to Business Edition
 
-Follow the below guides to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)** based on the type of your **Appsmith Installation:**
+Follow the steps below to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)**. Choose the type of  **Appsmith Installation** to get started.
 
-:::note
-You'll need your  [**Business Edition License Key**](https://www.appsmith.com/pricing)  to complete the upgrade.
+:::info License Key
+You would need a **Business Edition License Key** to complete the upgrade. Learn more on [**Business Edition & pricing**](https://www.appsmith.com/pricing) options.
 :::
 
-
-* [Docker](docker.md)
-
-
-:::note
-For **non-docker Appsmith installations**, please get in touch with us via [discord](https://discord.com/invite/rBTTVJp) or [email](mailto:support@appsmith.com). We will help you upgrade your Appsmith instance.
-:::
-
-## License Renewal
-
-The Appsmith server shuts down if your Business license key is expired. You will see an invalid license key message like below in the logs:
-
-```
+<div class="containerBorder">
+    <div class="containerGrid">
+        <div class="columnGrid column-one" align="center">
+            <div class="containerCol">
+                <a href="docker/">
+                <img class="containerImage" src="/img/docker-logo.png" alt="Docker-logo"/>
+                </a> 
+            </div> 
+            <b><a href="docker/">Docker</a></b>
+        </div>
+        <div class="columnGrid column-two" align="center">
+       </div>
+        <div class="columnGrid column-three" align="center">
+        </div>
+    </div>
+</div>
+
+## License renewal
+
+The Appsmith server shuts down once the license key expires. You would see `Invalid license key` message in the logs:
+
+```bash
 Exiting application. Invalid license key.
 ```
 
-:::info
-Please get in touch with our support team either on [Discord](https://discord.com/invite/rBTTVJp) or by [email](mailto:support@appsmith.com) to renew your license.
-:::
+Raise a request on [Discord](https://discord.com/invite/rBTTVJp), or [send an email to support](mailto:support@appsmith.com) to raise a renewal request. Once you have a renewed license key, [replace](docker#step2-add-or-replace-license-key) it to have Appsmith up and running.
+
 
-Once you have a new license key, [replace the existing license key](docker.md#step3-add-or-replace-license-key) in the `docker-compose.yml` file.
+## Next steps
```

Comment:
This is just a landing page, and hence the troubleshooting section is added to the docker upgrade page, as users will find it easily there. The next steps about managing and configuring are added here so that the users can quickly move to the section for configuring business features. We may choose to remove the Next steps section from the landing page and have them added to an individual page.

Reply:
On second thought, do we know the most wanted feature for which users are upgrading to Business Edition? We can add that to the next steps.

Reply:
How about we put all the business features that we have available today.

Reply:
Yes, that sounds good. I'll add them.

Reply:
Could you please share the link for Backup & Restore? I think it's https://docs.appsmith.com/advanced-concepts/more/backup-restore

Reply:
Yeah, I think it is this one too. But can we first confirm is this is Business edition only? @Rishabhkaul

Reply:
Ok I checked this with Pranay Agarwal. Quoting him below

> Hi Dancia, so even today all users can back-up their apps & data from the backend, but my understanding is that we were going to introduce some UI for customers/users to do this as well (in BE edition). But TM pod is not planning to pick this up anytime soon, so in our new pricing page also, which will go live in the next 3-4 days, this feature will not be present

So let's remove the Backup and Restore link altogether @jnikhila and keep just SSO and Audit logs